sql/moz-fx-data-shared-prod/telemetry_derived/crash_signatures_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Crash Signatures
+description: |-
+  Crash signatures generated by siggen from crash_symbolicated
+owners:
+- dthorn@mozilla.com
+labels:
+  application: firefox
+  schedule: daily
+scheduling:
+  dag_name: bqetl_crash
+  start_date: '2023-12-10'
+  retry_delay: 5m
+  gke_cluster_name: workloads-prod-v1
+  gke_location: us-west1
+  gcp_conn_id: google_cloud_airflow_gke
```

Comment:
Is all of this scheduling configuration just default config? If not, should all the queries added here use it?

Reply:
ah, it is default, it was copy-pasta from another place that received it when the default was changing